src/shims/manual/mscorlib.forwards.cs

```diff
@@ -27,6 +23,14 @@
 [assembly:System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Collections.Generic.LongEnumEqualityComparer<>))]
 [assembly:System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Collections.ListDictionaryInternal))]
 
+// Enum comparers that only exist in CoreCLR
```

Comment:
They should not be needed here at all. This file should be same between CoreCLR and CoreRT.